docs/USERAPI.md

```diff
@@ -0,0 +1,206 @@
+# User API documentation
+
+This is the documentation for the new user API (and only the new one) that was created in 2020.
+
+## Authorization
+
+### Description
+
+User authorization is been done with OAuth 2.0 and Google as the OAuth 2.0 provider. When calling the authentication URL you will
+be redirected to Google OAuth system (So the user need to be able to see the screen), after Google will authenticate the user he will be redirected
+to https://www.anyway.co.il/authorize/google/callback/google and from there the flow will be redirected to the URL that was given in
+the param `redirect_url` or to the default URL.
+
+### URL struct
```

Comment:
It is difficult to read such documentation for multiple calls - each request has Struct, method, etc.

I suggest using smaller titles, and less of them:
```
### Authorization (GET | https://www.anyway.co.il/authorize/google)
User authorization is been...
##### Examples:
https://www.anyway.co.il/authorize/google
##### Returns:
...

```

Reply:
However, I didn't see the MD parse result, only this PR, so you can ignore this comment if you like :)